serveradmin/servershell/static/js/servershell/command.js

```diff
@@ -62,7 +76,9 @@ servershell.update_attribute = function(object_id, attribute_id, new_value, mult
         let to_remove;
 
         // Don't add or remove empty values
-        new_value = new_value.filter(v => v.trim() !== '');
+        new_value = new_value.filter(function(v) {
```

Comment:
What was the reason to change the syntax from the arrow-function syntax?

Reply:
None